english/metallica/albums/albums.md

```diff
@@ -0,0 +1,13 @@
+Choose your album!
+
+1. [Kill 'Em All (1983)](debut/debut.md)
+
+2. [Ride The Lightning (1984)](second/second.md)
+
+3. [Master Of Puppets (1985)](third/third.md)
+
+4. [...And Justice For All(1989)](fourth/fourth.md)
+
+5. [The Black Album (1991)](eponymous/eponymous.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.